backend/pkg/k8cache/cacheInvalidation.go

```diff
@@ -87,3 +98,175 @@ func SkipWebSocket(r *http.Request, next http.Handler, w http.ResponseWriter) bo
 
 	return false
 }
+
+func contains(slice []string, str string) bool {
+	for _, v := range slice {
+		if v == str {
+			return true
+		}
+	}
+
+	return false
+}
+
+// returnGVRList return gvrList which is group, version, resource which is all the supported resources
+// that are supported by the k8s server.
+func returnGVRList(apiResourceLists []*metav1.APIResourceList) []schema.GroupVersionResource {
+	skipKinds := map[string]bool{
+		"Lease": true,
+		"Event": true,
+	}
+
+	var gvrList []schema.GroupVersionResource
+
+	for _, apiResource := range apiResourceLists {
+		for _, resource := range apiResource.APIResources {
+			if strings.Contains(resource.Name, "/") || skipKinds[resource.Kind] {
+				continue
+			}
+
+			if contains(resource.Verbs, "list") && contains(resource.Verbs, "watch") {
```

Comment:
Could delete your contains function, and use the go built in slices.Contains function. https://pkg.go.dev/slices#Contains
```go
if slices.Contains(resource.Verbs, "list") && slices.Contains(resource.Verbs, "watch") {
```